lightning/src/ln/chan_utils.rs

```diff
@@ -890,6 +890,129 @@ impl BuiltCommitmentTransaction {
 	}
 }
 
+/// This class tracks the per-transaction information needed to build a closing transaction and to
+/// actually build it and sign.
```

Comment:
"to build a closing transaction and to actually build it" is a bit redundant -- rephrase plz?

Reply:
This sentence structure is copied from the docs of `CommitmentTransaction`, so I'm changing it in both places.  I'm changing just a single word, please see if it makes more sense, or let me know if you have any other ideas.

Reply:
Looks good! thanks